### DISCUSSION TITLE
Why not a Preferences Panel?

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#287] [Ideas]
A **Preferences** panel is a good Idea for people wants to configure all. I know you want no configuration, but I think: we should have all configured for the **most** users, but: if the users wants to configure all, Why complicate it by putting few graphic configuration options and leaving the rest in files that many consider cryptic?

**Comment by probonopd** [comment]
We do not need a "panel" because in helloSystem, Preferences are just regular applications.
I think our system is much simpler, which is an advantage over having a "panel".
You can group our applications in folders as you like, can have open as many as you like at that same time, etc.
So I think our system is way better than what macOS, KDE, Gnome etc. have.

A "panel" as you are probably thinking of is not a UI concept I know from the [HIG](https://dl.acm.org/doi/book/10.5555/573097). The big step forward from System 6 to System 7 was that Control Panels (note the plural!) worked more like regular applications and could be organized in normal folders:

![image](https://user-images.githubusercontent.com/2480569/145903303-cdf625de-d070-4f5a-8fd4-b8f1cf29dc5d.png)

In this regard, I think Classic Mac OS was better than Mac OS X, which made things more complicated.
